CHANGELOG.md

```diff
@@ -0,0 +1,52 @@
+terraform state mv module.odc_eks.module.vpc.aws_route_table_association.private[0] module.odc_eks.module.vpc[0].aws_route_table_association.private[0]
+terraform state mv module.odc_eks.module.vpc.aws_route_table_association.private[1] module.odc_eks.module.vpc[0].aws_route_table_association.private[1]
+terraform state mv module.odc_eks.module.vpc.aws_route_table_association.private[2] module.odc_eks.module.vpc[0].aws_route_table_association.private[2]
+terraform state mv module.odc_eks.module.vpc.aws_route_table_association.public[0] module.odc_eks.module.vpc[0].aws_route_table_association.public[0]
+terraform state mv module.odc_eks.module.vpc.aws_route_table_association.public[1] module.odc_eks.module.vpc[0].aws_route_table_association.public[1]
+terraform state mv module.odc_eks.module.vpc.aws_route_table_association.public[2] module.odc_eks.module.vpc[0].aws_route_table_association.public[2]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.database[0] module.odc_eks.module.vpc[0].aws_subnet.database[0]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.database[1] module.odc_eks.module.vpc[0].aws_subnet.database[1]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.database[2] module.odc_eks.module.vpc[0].aws_subnet.database[2]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.private[0] module.odc_eks.module.vpc[0].aws_subnet.private[0]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.private[1] module.odc_eks.module.vpc[0].aws_subnet.private[1]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.private[2] module.odc_eks.module.vpc[0].aws_subnet.private[2]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.public[0] module.odc_eks.module.vpc[0].aws_subnet.public[0]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.public[1] module.odc_eks.module.vpc[0].aws_subnet.public[1]
+terraform state mv module.odc_eks.module.vpc.aws_subnet.public[2] module.odc_eks.module.vpc[0].aws_subnet.public[2]
+terraform state mv module.odc_eks.module.vpc.aws_vpc.this[0] module.odc_eks.module.vpc[0].aws_vpc.this[0]
+terraform state mv module.odc_eks.module.vpc.aws_vpc_endpoint.s3[0] module.odc_eks.module.vpc[0].aws_vpc_endpoint.s3[0]
+terraform state mv module.odc_eks.module.vpc.aws_vpc_endpoint_route_table_association.private_s3[0] module.odc_eks.module.vpc[0].aws_vpc_endpoint_route_table_association.private_s3[0]
+terraform state mv module.odc_eks.module.vpc.aws_vpc_endpoint_route_table_association.private_s3[1] module.odc_eks.module.vpc[0].aws_vpc_endpoint_route_table_association.private_s3[1]
+terraform state mv module.odc_eks.module.vpc.aws_vpc_endpoint_route_table_association.private_s3[2] module.odc_eks.module.vpc[0].aws_vpc_endpoint_route_table_association.private_s3[2]
+terraform state mv module.odc_eks.module.vpc.aws_vpc_endpoint_route_table_association.public_s3[0] module.odc_eks.module.vpc[0].aws_vpc_endpoint_route_table_association.public_s3[0]
+```
```

Comment:
`terraform state mv module.odc_eks.module.vpc module.odc_eks.module.vpc[0]`

Reply:
Thanks, missing one added.

Reply:
I meant that that one is the only one you need.